scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -535,23 +534,27 @@ def __init__(self, storage_name: str, container_name: str, container_path: str):
             None,
         )
 
-    @property
-    def uri(self) -> str:
-        """Asset URI."""
-        # If we have already cached the URI, then simply return it
-        if self._uri is not None:
-            return self._uri
+    def get_uri(self, token_expiration_timedelta=datetime.timedelta(hours=1)) -> str:

```

Comment:
Sorry, Kelly. What I meant was this, along with an accompanying import statement above:
```suggestion
    def get_uri(self, token_expiration: timedelta = timedelta(hours=1)) -> str:
```